kubernetes/adapter.go

```diff
@@ -304,6 +305,11 @@ func (a *Adapter) ListResources() ([]*Ingress, error) {
 	}
 	rgs, err := a.ListRoutegroups()
 	if err != nil {
+		log.Warnf("Failed to list RouteGroups: %v, to get more information https://opensource.zalando.com/skipper/kubernetes/routegroups/#routegroups", err)
```

Comment:
I'm afraid this will be quite spammy in the logs as it will happen every sync interval. IMO it should be ok to run without RouteGroup support and you don't want to see a lot of logs telling the same thing if you can't turn it off.

Either we can add a flag to disable route groups completely, or we can detect the first time we try to list routegroups and e.g. set a field `a.routeGroupSupport = false` and only log `if a.routeGroupSupport`. WDYT?

Reply:
From 9 to 10 lines per interval is too spammy?

Reply:
The problem with storing the value is that after CRD apply it does not work, but you would have to delete the pod to retry.

Reply:
You could still try to list the routes every interval (and reset the flag). I just think it doesn't bring any value to the user to log a warning that they can't do anything about (apart from deploying the CRD which they may not need/want).

Reply:
This makes sense!
I added as suggested the check